kok-api/src/main/java/com/kok/kokapi/room/adapter/in/web/RoomController.java

```diff
@@ -41,44 +43,63 @@ public ResponseEntity<ApiResponseDto<RoomDetailResponse>> getRoomDetail(@PathVar
     @Operation(summary = "약속방 생성", description = "새로운 약속방을 생성합니다.")
     @PostMapping("/rooms")
     public ResponseEntity<ApiResponseDto<RoomCreateResponse>> createRoom(@Valid @RequestBody CreateRoomRequest request) {
-        var host = new Member(request.hostNickname(), request.hostProfile(), MemberRole.LEADER);
-        var room = createRoomUseCase.createRoom(
+        String memberId = generateMemberId();
+        String nickname = request.hostNickname();
+        String profile = request.hostProfile();
+        Member host = new Member(memberId, nickname, profile, MemberRole.LEADER);
+
+        Room room = createRoomUseCase.createRoom(
                 request.roomName(),
                 request.capacity(),
                 host
         );
 
-        var response = RoomCreateResponse.from(room, 1, 1);
+        RoomCreateResponse response = RoomCreateResponse.from(room, 1, request.capacity() - 1);
```

Comment:
```suggestion
        RoomCreateResponse response = RoomCreateResponse.from(room, 1, room.getCapacity() - 1);
```
이왕이면 만들어진 도메인의 값을 사요하면 어떨까해서요 ㅎㅎ

Reply:
저도 동의합니다! 👍